atc/builds/planner.go

```diff
@@ -125,11 +126,12 @@ func (visitor *planVisitor) VisitPut(step *atc.PutStep) error {
 	resource.ApplySourceDefaults(visitor.resourceTypes)
 
 	atcPutPlan := atc.PutPlan{
-		Name:     logicalName,
-		Resource: resourceName,
-		Type:     resource.Type,
-		Source:   resource.Source,
-		Params:   step.Params,
+		Name:                 logicalName,
+		Resource:             resourceName,
+		Type:                 resource.Type,
+		Source:               resource.Source,
+		Params:               step.Params,
+		ExposeBuildCreatedBy: resource.ExposeBuildCreatedBy,
```

Comment:
Ideally this can also be removed once we have a `atc.NotifyPlan` (or something). Makes sense for now though.

Reply:
Good to know.